README.md

```diff
@@ -536,6 +542,7 @@ Want to join this illustrious group? Have a look at the **[contributing guidelin
 <!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
 <!-- prettier-ignore-start -->
 <!-- markdownlint-disable -->
+<div class = "table-container">
```

Comment:
Not sure if this is a good idea. The bot might overwrite his 🤔 

We should try using the config as suggested in the linked issue.

Reply:
As for the others, this won't be a problem right? I'll remove the class for contributors table. Moreover, Jekyll is very limiting and the allcontributors configurations gives very few options. even if we changed the `contributorsPerLine` to 3 (it is 4 now), the contributors tab will not change dynamically to screen size, but will show 3 per row for all screen sizes, which only looks awkward. I'll revert the ones in the contributors table.